include/swift/Driver/PrettyStackTrace.h

```diff
@@ -0,0 +1,76 @@
+//===--- PrettyStackTrace.h - PrettyStackTrace for the Driver ---*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
```

Comment:
It's 2018. Not sure how we're going to handle this across the whole project, though.